android/app/src/main/java/me/rainbow/MainActivity.java

```diff
@@ -16,6 +17,7 @@
   protected void onCreate(Bundle savedInstanceState) {
         SplashScreen.show(this, R.style.NoActionBar);
         super.onCreate(savedInstanceState);
+        WebView.setWebContentsDebuggingEnabled(true);
```

Comment:
what's this?

Reply:
To be able to debug the webview from chrome. Doesn't change anything for end users, but it's useful for us.